crates/rolldown/tests/esbuild/packagejson/package_json_browser_issue2002_b/artifacts.snap

```diff
@@ -24,7 +25,7 @@ source: crates/rolldown_testing/src/integration_test.rs
 
 //#region src/node_modules/pkg/sub/foo.js
 var require_foo = __commonJS({ "src/node_modules/pkg/sub/foo.js"() {
-	require("sub");
+	__require("sub");
```

Comment:
Here we just have different output compared to esbuild, esbuild did not have `__require`, 
```js
// Users/user/project/src/node_modules/pkg/sub/bar.js
var require_bar = __commonJS({
  "Users/user/project/src/node_modules/pkg/sub/bar.js"() {
    works();
  }
});

// Users/user/project/src/node_modules/pkg/sub/foo.js
var require_foo = __commonJS({
  "Users/user/project/src/node_modules/pkg/sub/foo.js"() {
    require_bar();
  }
});

// Users/user/project/src/entry.js
require_foo();
```